docs/source/upcoming_release_notes/614-LAMP_motion_system_class.rst

```diff
@@ -0,0 +1,33 @@
+614 LAMP motion system class
+#################
```

Comment:
Oh, and this annoying thing about sphinx/restructured text (have to be the same length):
```suggestion
############################
```

Reply:
Ah of course, resolved in commit 5d3e726